### PR TITLE
Dept 47 update readme

### PR DESCRIPTION
**Pull request by blamboy090**
Added more details to ReadMe

### CODE REVIEW

**Review by Incarnate-Infinity — COMMENTED** [review]
I think this is looking good. I really like the gif demonstration!

I might recommend fleshing out a couple of the portions of the readme:

- While it should be clearly visible, it may be worthwhile to note that upon running tests a coverage report will be generated in the console where the command is run.
- We should probably mention somewhere that the application requires a user to login with a DEPT authenticated email to function

File: README.md [hunk 1]
```diff
@@ -1,12 +1,35 @@
...
 - Install dependencies
   `yarn install`
-- Reach out to Nat Ring to get local environment variables. Copy `.env.example` and rename it as `.env.local` and include those values in this file.
 - Run the dev server
   `yarn dev`
-- This project uses SASS and will need a compiler. We recommend the Live SASS Compiler plugin for VS Code. This will compile the `.scss` files and generate the `.css` files which will be read by the other program files for styling.
 
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+
+Please advise a valid DEPT authenticated email is required to access the application in full.
```
Comment (Incarnate-Infinity) [inline README.md:22]: Not sure if this is the (or a) proper phrasing for this. Suggest adjusting to "Please be advised that...". If this actually is proper/legitimate phrasing for this please ignore.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,35 @@
...
+
+## Testing
+
+- You may run tests with coverage via `yarn test`
+- Coverage report will be present in console where command is ran
```
Comment (Incarnate-Infinity) [inline README.md:27]: Bit of phrasing here on the tenses. Rather than "... command is ran" I believe "... command was ran".

**Review by Incarnate-Infinity — CHANGES_REQUESTED** [review]
I appreciate the additions! Couple of grammar/wording changes I'd recommend and then I think this is good to go.

File: README.md (hunk 2 above)
Comment (blamboy090) [reply README.md]: updated

File: README.md (hunk 1 above)
Comment (blamboy090) [reply README.md]: updated

**Review by Incarnate-Infinity — APPROVED** [review]
LGTM 👍

**Review by lsanchezcode — APPROVED** [review]
LGTM! :)

**Pull request merged**